src/com/tony/server/response/LogResponse.java

```diff
@@ -0,0 +1,21 @@
+package com.tony.server.response;
+
+import com.tony.server.Logger;
+
+import java.util.HashMap;
+
+public class LogResponse extends Response{
+    private final Logger logger;
+
+    public byte[] respond(){
+        setBody(logger.getLog());
+        return ResponseBuilder.buildResponse(getStatusLine(),
+                getHeaders(), getBody()).getBytes();
+    } 
+    
+    public LogResponse(Logger logger) {
+        this.logger = logger;
+        setStatusLine("HTTP/1.1 200 OK");
```

Comment:
Also, not 100% related to this diff, it seems like you have a lot of repetition when it comes to setting the status line and headers of your response. 

A status line is `HTTP-Version SP Status-Code SP Reason-Phrase CRLF`:
  HTTP-Version = "HTTP/1.1"
  Status-Code = "200" in this case
  Reason-Phrase = "OK" in this case
  CRLF = `\r\n`

Since this is the shared format of your status line, it seems like you should be pulling these commonalities into your response builder, not only to be DRY, but to make it easier to understand. 

Another thing I would suggest for this, is to keep all of your Status-Codes and Response-Phrases in one place as constants or something, and reference them in your response, so you don't have to keep building these strings in every response.